pkgs/tools/misc/pywb/default.nix

```diff
@@ -0,0 +1,207 @@
+{ lib, fetchFromGitHub, python39, stdenv }:
```

Comment:
is 3.9 a hard requirement? 

```suggestion
{ lib, fetchFromGitHub, python3, stdenv }:
```

Reply:
If you use the `callPackage` trick below:

```suggestion
{ lib, fetchFromGitHub, python, stdenv }:
```

Reply:
It shouldn't be a hard requirement.